src/main/java/org/sagebionetworks/web/shared/asynch/AsynchType.java

```diff
@@ -30,12 +38,11 @@
 
 public enum AsynchType implements IsSerializable {
 	TableAppendRowSet(TABLE_APPEND, AppendableRowSetRequest.class, RowReferenceSetResults.class), TableQuery(TABLE_QUERY, QueryBundleRequest.class, QueryResultBundle.class), TableQueryNextPage(TABLE_QUERY_NEXTPAGE, QueryNextPageToken.class, QueryResult.class), TableCSVUpload(TABLE_UPLOAD_CSV, UploadToTableRequest.class, UploadToTableResult.class), TableCSVUploadPreview(TABLE_UPLOAD_CSV_PREVIEW, UploadToTablePreviewRequest.class, UploadToTablePreviewResult.class), TableCSVDownload(TABLE_DOWNLOAD_CSV, DownloadFromTableRequest.class, DownloadFromTableResult.class), BulkFileDownload(FILE_BULK, BulkFileDownloadRequest.class, BulkFileDownloadResponse.class), TableTransaction(TABLE_TRANSACTION, TableUpdateTransactionRequest.class, TableUpdateTransactionResponse.class), Doi(DOI, DoiRequest.class, DoiResponse.class), AddFileToDownloadList(DOWNLOAD_LIST_ADD, AddFileToDownloadListRequest.class, AddFileToDownloadListResponse.class), DownloadStorageReport(STORAGE_REPORT,
-			DownloadStorageReportRequest.class, DownloadStorageReportResponse.class), CreateJsonSchema(SCHEMA_TYPE_CREATE, CreateSchemaRequest.class, CreateSchemaResponse.class), ViewColumnModelRequest(VIEW_COLUMN_MODEL_REQUEST, ViewColumnModelRequest.class, ViewColumnModelResponse.class), GetValidationSchema(SCHEMA_TYPE_VALIDATION, GetValidationSchemaRequest.class, GetValidationSchemaResponse.class), QueryDownloadList(DOWNLOAD_LIST_V2, DownloadListQueryRequest.class, DownloadListQueryResponse.class);
-
+			DownloadStorageReportRequest.class, DownloadStorageReportResponse.class), CreateJsonSchema(SCHEMA_TYPE_CREATE, CreateSchemaRequest.class, CreateSchemaResponse.class), ViewColumnModelRequest(VIEW_COLUMN_MODEL_REQUEST, ViewColumnModelRequest.class, ViewColumnModelResponse.class), GetValidationSchema(SCHEMA_TYPE_VALIDATION, GetValidationSchemaRequest.class, GetValidationSchemaResponse.class), QueryDownloadList(DOWNLOAD_LIST_V2, DownloadListQueryRequest.class, DownloadListQueryResponse.class), AddToDownloadList(DOWNLOAD_LIST_ADD_V2, AddToDownloadListRequest.class, AddToDownloadListResponse.class), DownloadPackageList(DOWNLOAD_LIST_PACKAGE_V2, DownloadListPackageRequest.class, DownloadListPackageResponse.class), DownloadListManifest(DOWNLOAD_LIST_MANIFEST_V2, DownloadListManifestRequest.class, DownloadListManifestResponse.class), FileHandleRestore(FILE_HANDLE_RESTORE, FileHandleRestoreRequest.class, FileHandleRestoreResponse.class);

```

Comment:
This fixes the build (there's a test that verifies that we have client-side representations for all Async job types available in the backend, which we didn't!)